bulbs/content/models.py

```diff
@@ -326,7 +327,9 @@ def save(self, *args, **kwargs):
             if kwargs is None:
                 kwargs = {}
             kwargs["index"] = False
-        return super(Content, self).save(*args, **kwargs)
+        content = super(Content, self).save(*args, **kwargs)
+        index_content_contributions.delay(self.id)
```

Comment:
no. That fixed the problem locally. However, the signal is not being recognized on test. After debugging I decided to move it into the save function because this needs to be out the door and this is seemingly functional.